assets/themes/campersites/campersites.json

```diff
@@ -0,0 +1,718 @@
+          "freeform": {
+            "key": "charge"
+          },
+          "condition": {
+            "and": [
+              "fee=yes"
+            ]
+          }
+        },
+        {
+          "render": {
+            "en": "This place is called {name}"
+          },
+          "question": {
+            "en": "What is this place called?"
+          },
+          "freeform": {
+            "key": "name"
+          },
+          "condition": {
+            "and": [
+              "name="
+            ]
+          }
+        },
+        {
+          "render": {
+            "en": "Sanitary dump station"
+          },
+          "question": {
+            "en": "Does this place have a sanitary dump station?"
+          },
+          "freeform": {
+            "key": "",
+            "addExtraTags": []
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "sanitary_dump_station=yes"
+                ]
+              },
+              "then": {
+                "en": "This place has a sanitary dump station"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "sanitary_dump_station=no"
+                ]
+              },
+              "then": {
+                "en": "This place does not have a sanitary dump station"
+              }
+            }
+          ]
+        },
+        {
+          "render": {
+            "en": "This place is operated by {operator}"
+          },
+          "question": {
+            "en": "Who operates this place?"
+          },
+          "freeform": {
+            "key": "operator"
+          }
+        },
+        {
+          "render": {
+            "en": "{capacity} campers can use this place at the same time"
+          },
+          "question": {
+            "en": "How many campers can stay here? (skip if there is no obvious number of spaces or allowed vehicles)"
+          },
+          "freeform": {
+            "key": "capacity",
+            "type": "pnat"
+          }
+        },
+        {
+          "render": {
+            "en": "internet access"
+          },
+          "question": {
+            "en": "Does this place provide internet access?"
+          },
+          "mappings": [
+            {
+              "if": {
+                "and": [
+                  "internet_access=yes"
+                ]
+              },
+              "then": {
+                "en": "There is internet access"
+              }
+            },
+            {
+              "if": {
+                "and": [
+                  "internet_access=no"
+                ]
+              },
+              "then": {
+                "en": "There is no internet access"
+              }
+            }
+          ],
+          "freeform": {
+            "key": "internet_access"
+          }
+        },
+        {
+          "render": {
+            "en": "internet access price"
```

Comment:
A rendering without freeform doesn't make sense - please remove it

Reply:
removed